src/entities/Reserved.students.entities.ts

```diff
@@ -0,0 +1,18 @@
+import {Entity, PrimaryGeneratedColumn, ManyToOne, JoinTable, JoinColumn, OneToOne} from 'typeorm';
+import {StudentsEntity} from "./Student.entity";
+import {RecruiterEntity} from "./Recruiters.entity";
+
+@Entity({name: 'reserved_students'})
+
+export class ReservedStudentsEntities {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @OneToOne(type => StudentsEntity)
```

Comment:
students nie powinno byc tablica? nie znam sie na relacjach one to one ale to wygląda jakby 1 rekruter mogł mieć 1 studenta

Reply:
tak createAt koniecznie, napisałam nawet reservedAt - co lepiej odda cel i załatwi nam odliczanie 10 dni.